exec/java-exec/src/main/java/org/apache/drill/exec/ops/FragmentContext.java

```diff
@@ -465,6 +441,14 @@ public Executor getExecutor(){
     return context.getExecutor();
   }
 
+  public long getFragmentMemoryLimit() {
+    return allocator.getLimit();
+  }
+
+  public void setFragmentMemoryLimit(long value) {
```

Comment:
getFragmentMemoryLimit and setFragmentMemoryLimit don't look like they are used anywhere. Let's just remove them.